content/concepts/authentication/content.md

```diff
@@ -175,3 +175,29 @@ podIdentity:
 ```
 
 Azure AD Pod Identity will give access to containers with a defined label for `aadpodidbinding`.  You can set this label on the KEDA operator deployment.  This can be done for you during deployment with Helm with `--set aadPodIdentity={your-label-name}`.
+
+#### Kiam Pod Identity for AWS
+
+[**Kiam**](https://github.com/uswitch/kiam/) lets you bind an AWS IAM Role to a pod using an annotation on the pod.
+
+You can tell KEDA to use Kiam via `podIdentity.provider`.
+
+```yaml
+podIdentity:
+  provider: kiam # Optional. Default: false
+```
+
+Kiam will give access to containers based on the role provided by the `iam.amazonaws.com/role` annotation on the pod.
```

Comment:
Thinking out loud - Since this is documentation on how to configure it for our dependency, should we document this here or merely point to their docs? For example, what if they change their approach?

Reply:
I do not expect kiam to change. I assume it will be superseded, rather than changed.

Reply:
I can remove these lines though. I understand the skepticism of essentially documenting another project.

Reply:
Maybe just add a link instead, but I think you already did it somewhere else, right?

Reply:
It's in the first paragraph. I will remove thsese sections.